src/PharoThings-Devices-HD44780.package/PotLCDHD44780Gpio.class/instance/write.mode..st

```diff
@@ -3,7 +3,7 @@ write: anHex mode: aBit
 	"mode:0 to send commands or mode:1 to send characters. Use writeCommand: and writeCharacter:"
 	| j |
 	j := 1.
-	1 milliSeconds wait.
+	999999 nanoSeconds wait.
```

Comment:
Can you comment this?

Reply:
I was using 1 miliSecond and has a big delay to write the characteres in display. I searched a microSeconds method and didn't find it, so I'm using the nanoSeconds. This 999999 is the limit, if I use 1000000 nanoSeconds has a delay also.

Reply:
I will use "Delay forSeconds: 0.000001"... Tomorrow I test it in the hardware.

Reply:
I am curious what happens when there is no delay at all?

Reply:
The chipset has an execution time to each command +- 37us. The "clear command" is slower with 1.52ms. And this time can change depends on the frequency of commands sent to it. 

In the tests I did, nothing happened. But to prevent an overflow when the frequency becomes high, I put this delay here and in the methods to set the parameters at the chipset (setDisplayControl: setDisplayFunction: and setDisplayMode:). They are imperceptible to human eyes.  

This is more clear on page 24 of chipset datasheet:  https://www.sparkfun.com/datasheets/LCD/HD44780.pdf